.github/workflows/main.yml

```diff
@@ -8,16 +8,53 @@ on:
   pull_request:
     branches: [ master ]
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+
+env:
+  CI: true
```

Comment:
**annotation**: This is temporary until we get the whole test suite refactored. For now we skip the image build step in the test script since this pipeline ensures that the correct images are available.